new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -148,6 +146,39 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
                 .name(serviceAccountName))));
     logger.info("Created service account: {0}", serviceAccountName);
 
+    // get Operator image name
+    String operatorImage = getOperatorImageName();
+    assertFalse(operatorImage.isEmpty(), "Operator image name can not be empty");
+    logger.info("Operator image name {0}", operatorImage);
+
+    // Create docker registry secret in the operator namespace to pull the image from repository
+    if (System.getenv("REPO_REGISTRY") != null && System.getenv("REPO_USERNAME") != null
+        && System.getenv("REPO_PASSWORD") != null && System.getenv("REPO_EMAIL") != null) {
+      repoRegistry = System.getenv("REPO_REGISTRY");
```

Comment:
This is a little weird in that it doesn't take the value of any variable unless all are set.  You could check each value in line like this:
`repoRegistry = Optional.ofNullable(System.getenv("REPO_REGISTRY")).orElse("dummy");`

Reply:
done